assets/constants.ts

```diff
@@ -5733,6 +5733,8 @@ export default {
     "LIST": "list"
   },
 
+  "EMAIL_REGEX": "^[^\\s@]+@[^\\s@]+\\.[^\\s@]+",
+
```

Comment:
Why is a different constant used here and in _profile.py_?

Reply:
Done, using this in profile.py